src/libraries/System.Globalization/tests/CompareInfo/CompareInfoTests.Compare.cs

```diff
@@ -13,6 +13,16 @@ public class CompareInfoCompareTests
         private static CompareInfo s_currentCompare = CultureInfo.CurrentCulture.CompareInfo;
         private static CompareInfo s_hungarianCompare = new CultureInfo("hu-HU").CompareInfo;
         private static CompareInfo s_turkishCompare = new CultureInfo("tr-TR").CompareInfo;
+        private static CompareInfo s_japaneseCompare = new CultureInfo("ja-JP").CompareInfo;
+        private static CompareOptions supportedIgnoreNonSpaceOption =
+            PlatformDetection.IsHybridGlobalization ?
```

Comment:
Here will be better to mention for Wasi/Browser. For now there is no implementation for OSX platforms but once native functions will be added for hybrid mode this needs to be changed.

Reply:
Thanks, changing to `IsHybridGlobalizationOnWasm`.